auslib/test/common/test_releases.py

```diff
@@ -0,0 +1,61 @@
+import json
+from auslib.test.common.base import CommonTestBase
+
+
+class TestPublicReleasesAPI(CommonTestBase):
+    def test_get_releases(self):
+        ret = self.public_client.get("/api/v1/releases")
+        got = json.loads(ret.data)
+        self.assertEquals(len(got["releases"]), 3)
+        self.assertIsInstance(got["releases"][0], dict)
```

Comment:
You don't need to fully verify the blobs, but verifying a few things on it instead of just the length and type would be good. In particular, the name, schema_version, and somthing in the platform and/or locale section. This will make the tests stronger, and guard against false positives if some of the underlying test data changes. Please do this for all tests that contain blobs.